Source/DafnyCore/AST/TypeDeclarations/TopLevelDecl.cs

```diff
@@ -1,10 +1,11 @@
 using System.Collections.Generic;
 using System.Diagnostics.Contracts;
 using System.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
```

Comment:
I'm a little sad to have the `DafnyCore` package depend on LSP-specific stuff, but I think the tradeoff is probably worth it.

Reply:
Looking more closely, I see that isn't a new dependency, so it's even more on the side of being worth it.

Reply:
I think it's to be expected that a compiler will have a public interface so it can be used as a library, and LSP happens to already contain many types that are useful for this interface, such as `Position` and `Range`. I've made this PR to reduce what's in scope from DafnyCore: https://github.com/dafny-lang/dafny/pull/5502